app/js/components/orderEntry/OrderEntryPage.jsx

```diff
@@ -43,6 +48,22 @@ export class OrderEntryPage extends PureComponent {
       errorMessage,
       labOrderData,
     } = this.props.createOrderReducer;
+    const {
+      orderables,
+    } = this.props.labOrderableReducer;
+    const { page } = this.state;
+    const {
+      labOrderableReducer,
+    } = prevProps;
+
+    if (labOrderableReducer.orderables[0].uuid !== orderables[0].uuid) {
+      if (page === 'Lab Orders') {
```

Comment:
Since this is going to be passed in as a url parameter, let's avoid spaces and capitalization, ie just use "lab" and "drug" instead of "Lab Order" and "Drug order"